.gitignore

```diff
@@ -125,3 +125,4 @@ packages/dd-trace/test/appsec/next/*/package.json
 packages/dd-trace/test/appsec/next/*/node_modules
 packages/dd-trace/test/appsec/next/*/yarn.lock
 !packages/dd-trace/**/telemetry/logs
+packages/datadog-plugin-azure-functions/test/integration-test/fixtures/node_modules
```

Comment:
I think this should already be the case by default?

Reply:
Yes but this line overrides that default and includes `node_modules` for some test directories

https://github.com/DataDog/dd-trace-js/blob/6b405c7754ba5f67fa7620fa89e6e6d40c18fbfc/.gitignore#L120